samples/basic/shell/src/ModuleErrorBoundary.tsx

```diff
@@ -11,18 +12,26 @@ function getErrorMessage(error: unknown) {
         : JSON.stringify(error);
 }
 
-export default function ModuleErrorBoundary() {
+export function ModuleErrorBoundary() {
     const error = useRouteError();
     const location = useLocation();
     const logger = useLogger();
 
-    logger.error(`[sample] An unmanaged error occurred while rendering the route with path ${location.pathname}`, error);
+    const handleReloadButtonClick = useCallback(() => {
+        window.location.reload();
+    }, []);
+
+    logger.error(`[shell] An unmanaged error occurred while rendering the route with path ${location.pathname}`, error);
```

Comment:
it feels like this should be in a useEffect. I agree that it works like this also because there is most likely not a rerender when it's displayed, but it's still the best way to do it